arduino-ide-extension/src/browser/arduino-preferences.ts

```diff
@@ -258,6 +266,17 @@ export const ArduinoConfigSchema: PreferenceSchema = {
       ),
       default: undefined,
     },
+    'arduino.monitor.dockArea': {
```

Comment:
```suggestion
    'arduino.monitor.dockPanel': {
```

Reply:
Excellent catch. The compiler should find such errors, so I [added](https://github.com/arduino/arduino-ide/pull/2102/files#diff-9ffaadbfa5cea6de54ac86a7b454b1217c95e811a98186c11af35c783f59015dR57-R61) a tiny TypeScript trick to let the compiler error when there is a typo next time.